web/packages/teleterm/src/ui/Search/useSearch.ts

```diff
@@ -83,7 +83,7 @@ export function useResourceSearch() {
           resourceTypeSearchFilter
         )
       );
-      const results = (await Promise.all(searchPromises)).flat().slice(0, 10);
```

Comment:
Could you add a test for `useResourceSearch` which verifies that it doesn't limit the number of results? If we reintroduced this `.slice` here, the test for `sortAndLimitResults` wouldn't catch the regression.

Alternatively we could do an integration test through `SearchBar` to verify that the whole component behaves correctly.

Reply:
Added a test for `useResourceSearch`.